src/main/webapp/js/script.js

```diff
@@ -130,11 +133,35 @@ async function getVersionHistory(platform, sdkName) {
   createNode("Owner: " + owner, "#owner");
 }
 
+// Fetch favorite SDKS
+async function getFavoriteSdks(userID) {
+  const response = await fetch(`v1/users/${userID}/sdks`);
+  const favorites = await response.json();
+  // Build the links for favorite sdks
+  const favoriteSDK = document.getElementById('favorites');  
+  for (i = 0; i < favorites.length; i++) {
+      favoriteSDK.appendChild(
+          createListElement(favorites[i]));
+      favoriteSDK.appendChild(document.createElement("HR"));
+      console.log(favorites[i]);
```

Comment:
nit: You should log some additional prefix to indicate that the log messages are favorite SDKs. Maybe add an additional console message before the for loop.